pysmarthashtag/api/authentication.py

```
@@ -390,6 +496,381 @@ async def async_auth_flow(self, request: Request) -> AsyncGenerator[Request, Res
+            "Xs-Ui": secrets.token_hex(16),
+            "Xs-Auth-Token": auth_token,
+            "Xs-Client-Id": self.client_id,
+            "Accept": "*/*",
+            "Accept-Language": "en-AU;q=1",
+            "Content-Type": "application/json",
+            "Cache-Control": "no-cache",
+        }
+
+        return headers
+
+    def _generate_vehicle_api_headers(self, body: str = None) -> dict[str, str]:
```

Ruff (0.15.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)